ynput/ayon-core
# Max: Fix the malfunctioning issue in Max Scene loader

## Changelog Description
Fix https://github.com/ynput/ayon-core/issues/152

## Additional info
Please test with different loaders.

## Testing notes:
1. Launch Max
2. Load the asset with the same name
3. Load the same asset again
4. And set versions for the assets
5. Should update and load accordingly.
